medusa/indexers/tmdb/tmdb.py

```diff
@@ -560,7 +560,7 @@ def get_last_updated_series(self, from_time, weeks=1, filter_show_list=None):
                     new_list.append(show.indexerid)
             total_updates = new_list
 
-        return total_updates
+        return list(total_updates)
```

Comment:
Reason is because this method should return a list. And a set() although it is also an iterable is not a list. You should always return what you advertise.